docs/source/en/quantization/bitsandbytes.md

```diff
@@ -200,7 +173,7 @@ model_8bit = AutoModelForCausalLM.from_pretrained(
 
 An "outlier" is a hidden state value greater than a certain threshold, and these values are computed in fp16. While the values are usually normally distributed ([-3.5, 3.5]), this distribution can be very different for large models ([-60, 6] or [6, 60]). 8-bit quantization works well for values ~5, but beyond that, there is a significant performance penalty. A good default threshold value is 6, but a lower threshold may be needed for more unstable models (small models or finetuning).
 
-To find the best threshold for your model, we recommend experimenting with the `llm_int8_threshold` parameter in [`BitsAndBytesConfig`]:
+To find the best threshold for your model, experiment with the `llm_int8_threshold` parameter in [`BitsAndBytesConfig`].
```

Comment:
This may be a good place for us to indicate that if you set the threshold to `0.0`, you'll get a significant improvement in inference speed, but with some potential accuracy trade-offs.